### DISCUSSION TITLE
Expected life

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#58] [Shipped - Live on production]
To manage the inventory risks it would be nice to have a way to compare age of the device against the expected life of the device. Is there currently a way to enter the expected life for a certain device model/type? So that we can compare age of a device against expected life and identify items due for replacement. 

It would also be great to then have a field to nominate devices for replacement (i think i made a separate request for that already).

**Comment by YvesR** [comment]
Isn't this the deprecation in years? At least from accounting perspective.

**Comment by BioMedGPH** [comment]
in the west we replace by budget, so by set economic life/budget. In low resource settings you run more by technical life, so it breaks and cannot be repaired or it starts to have too many problems. Generally this will be when you pass beyond expected technical life (bath tub curve starts to rise). This is generally given by manufacturers or from public sources. It would be great to have this number in Samedis and flag devices getting near or beyond expected service life.

Deprecation could be used for this of course, but as you indicated that is more from an economic perspective. This might depend on the actual price you paid for the device and its 'profit'. For technical life however this is more inherent to the design, and thus this might be a property of a certain device type/model, and not of a certain inventory item. So the Expected Technical Life field should be placed in the Device Type.

**Comment by BioMedGPH** [comment]
In addition to this, the technical life should be a property of the device type in my opinion, that would apply to all inventory items of that type. Is there any plan to add this?
